spt.c

```diff
@@ -131,8 +132,14 @@ main(int argc, char *argv[])
 
 	for (i = 0; ; i = (i + 1) % LEN(timers)) {
 		notify_send(timers[i].cmt);
-		for (timecount = 0; timecount < timers[i].tmr; timecount += inc)
-			sleep(1);
+		timecount = 0;
+		while (timecount < timers[i].tmr)
```

Comment:
I think these lines needs some improvement. I can't think of any better way to rework the logic.